### ISSUE TITLE
Command class class diagram

### ISSUE THREAD

**Issue by azriellee** [azriellee/pe#24]
![image.png](https://raw.githubusercontent.com/azriellee/pe/main/files/fc548afb-ae63-443d-aafc-e47b54447771.png)
Could consider drawing a partial class diagram instead? as the current class diagram might be too wordy and in depth

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
We wanted it to be wordy in order to give the developer a better understanding. It's better to be wordy in a DG.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
